### PR TITLE
Homework 6

### PR DESCRIPTION
**Pull request by 0Lyova0**
Hi!

Home work 6 done.

### CODE REVIEW

File: HW6/task_1_2.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+def add_numbers(a, b):
+    return a ** b
+
+exp_numbers = add_numbers(2, 3)
+print(exp_numbers)
+
+
+def add_numbers2(a=2, b=5):
```
Comment (acman) [inline HW6/task_1_2.py]: task is to path any number of arguments, `*args` should be used there

**Review by acman — CHANGES_REQUESTED** [review]
Almost there, several changes should be submitted

**Comment by 0Lyova0** [comment]
Hi!

changes are made

File: HW6/task_1_2.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# I don't know how to_do 2 ** 0.5
```
Comment (acman) [inline HW6/task_1_2.py:1]: The first function was ok. My comment was addressed only to the second task

**Review by acman — APPROVED** [review]
👍

**Pull request merged**